spec/libraries/coursemology_docker_container_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CoursemologyDockerContainer do
```

Comment:
won't fix. these are all the tests that will provide decent coverage of the main functions.